demo/multi_ledger_config.yml

```diff
@@ -0,0 +1,6 @@
+- id: localVON
+  is_production: true
+  genesis_url: 'http://localhost:9000/genesis'
```

Comment:
This fails when running `./run_demo faber --multi-ledger` as the default von-network runs in docker, as does the faber demo (`ConnectionRefusedError: [Errno 111] Connect call failed ('127.0.0.1', 9000)`)

This needs to use the docker IP or host name to connect.